packages/services/f-services/CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v1.11.0
+------------------------------
+*May 5, 2021*
+
```

Comment:
Do the date and the version need to be updated here?

Reply:
Typical that I've updated it locally and forgot to push. 👀